lib/facebook.ex

```diff
@@ -69,8 +69,10 @@ defmodule Facebook do
   """
   @spec me(fields, access_token) :: response
   def me(fields, access_token) do
-    if !is_nil(Config.appsecret) do
-      fields = fields ++ [appsecret_proof: encrypt(access_token)]
+    fields = if !is_nil(Config.appsecret) do
+      fields ++ [appsecret_proof: encrypt(access_token)]
+    else
+      fields
```

Comment:
while you're at it.. could you make that a separate function and call that everywhere? :D